features/publisher.feature

```diff
@@ -7,3 +7,22 @@ Feature: (Mainstream) Publisher
     When I request "/healthcheck/ready"
     Then JSON is returned
     And I should see ""status":"ok""
+
+  @replatformingnotproduction @notproduction
```

Comment:
Mentioned in https://github.com/alphagov/govuk-infrastructure/pull/329 - I'm not sure we'll need `replatformingnotproduction` given we have the `notproduction` tag.

Reply:
Noted.